### ISSUE TITLE
Documents should have a wrapper element, to make it extendable in the future, without breaking the format

### ISSUE THREAD

**Issue by jasollien** [buildingSMART/BCF-XML#255]
Currently documents is like this:
```
<documents>
  <document />
  <document />
<documents>
```

In order to make it extenable in the future I suggest something like this:

```
<documentInfo>
  <documents>
    <document />
    <document />
  <documents>
</documentInfo>
```

Then we are free to add the props we would like to documentInfo in the future. 

example:
```
<documentInfo>
  <someDocumentConfiguration />
  <documents>
    <document />
    <document />
  <documents>
</documentInfo>
```
Maybe there are better names than "documentInfo"?

**Comment by jasollien** [comment]
@pasi-paasiala, @ykulbak: Any inputs to this?

**Comment by cschladetsch** [comment]
Maybe documentMeta? It is a container for meta-information about documents, no?

"Info" seems too generic, which perhaps smelled bad to you in the first place. As you know, Info implies direct data, whereas Meta means info about data.

**Comment by jasollien** [comment]
documentMeta is fine by me. I'll create a suggestion

**Comment by ykulbak** [comment]
closed by #256
